src/clique_table.erl

```diff
@@ -91,7 +91,9 @@ resize_row(MaxLength, Widths) ->
     Sum = lists:sum(Widths),
     case Sum > MaxLength of
 	true ->
-	    resize_items(Sum, MaxLength, Widths);
+	    NewWidths = resize_items(Sum, MaxLength, Widths),
+	    % go around again, just in case some columns were unshrinkable
+	    resize_row(MaxLength, NewWidths);
```

Comment:
I haven't chased down exactly why, but at 93 columns wide, your sample data gives me an infinite loop, and I'm assuming this is where part of the problem lies.